PatientWebApplication/SearchMicroserviceApi/Model/Prescription.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+
+namespace SearchMicroserviceApi.Model
+{
+    public class Prescription : Entity
+    {
+        public int patientsid { get; set; }
+        public virtual List<Medicine> Medicines { get; set; }
+        public bool isUsed { get; set; }
+        public string comment { get; set; }
```

Comment:
Write class properties to start with upper letter and remove "this" in constructors